requirements.txt

```diff
@@ -5,3 +5,7 @@ numpy>1.7,<1.15; python_version == '2.7'
 numpy>1.7; python_version >= '3.4'
 ## currently conflicts if numpy isn't installed, as it will download it's own version due to steup_requries
 root_numpy
+## 'tabulate' dropped explicit support for Python 2.6 in version 0.8.1
+## Version 0.8.0 isn't in the CHANGELOG
+tabulate>=0.4.4,<0.8; python_version == '2.6'
```

Comment:
In one case you restrict `tabulate` to not be higher than `0.7.7` but here it's less than `0.8`.  Shouldn't these match?

Reply:
They do match (`next(0.7.7) == 0.8` for `tabulate`). I'll make it more obvious